PresentScreenings/Table Classes/FilmOutlineLevel.cs

```diff
@@ -93,5 +93,13 @@ public void SetInfo(NSTextField view)
             }
         }
         #endregion
+
+        #region Public Methods
+        public static bool ScreeningIsSelected(Screening screening)
+        {
+            AppDelegate app = (AppDelegate)NSApplication.SharedApplication.Delegate;
+            return app.Controller.CurrentScreening == screening;
+        }
```

Comment:
Please refrain from out-of-scope changes such as this one.
But having done this, please consider to draw the on-demand screening window as well.

Reply:
Done.